### PR TITLE
Caddy server configuration

### PR DESCRIPTION
**Pull request by Rurik19**
English and french languages. See #35

### CODE REVIEW

File: fr/admin/installation/virtualhosts.md [hunk 1]
```diff
@@ -160,3 +160,25 @@ url.rewrite-if-not-file = (
...
+```caddy
+yourdomain.ru {
+  root /var/www/wallabag/web
+  fastcgi / /var/run/php7-fpm.sock php {
+    index app.php
+  }
+  rewrite / {
+    to {path} {path}/ /app.php?{query}
+  }
+  tls your@emai.ru
```
Comment (tcitworld) [inline fr/admin/installation/virtualhosts.md]: `email` ?

File: en/admin/installation/virtualhosts.md [hunk 2]
```diff
@@ -158,3 +159,24 @@ url.rewrite-if-not-file = (
...
+```caddy
+yourdomain.ru {
+  root /var/www/wallabag/web
+  fastcgi / /var/run/php7-fpm.sock php {
+    index app.php
+  }
+  rewrite / {
+    to {path} {path}/ /app.php?{query}
+  }
+  tls your@emai.ru
```
Comment (tcitworld) [inline en/admin/installation/virtualhosts.md]: `email` ?

Comment (Rurik19) [inline en/admin/installation/virtualhosts.md]: from documentation

> email is the email address to use with which to generate a certificate with a trusted CA. By providing an email here you will not be prompted when you run Caddy.

Comment (tcitworld) [inline en/admin/installation/virtualhosts.md]: I meant `@email.ru` instead of `@emai.ru`.

Comment (Rurik19) [inline en/admin/installation/virtualhosts.md]: of course. made commit

**Comment by Rurik19** [comment]
I made something wrong with commit? Rewiev now 'slate'

**Comment by j0k3r** [comment]
@wallabag/translators hey guys, we need the :it: and :de: translations.

**Comment by matteocoder** [comment]
@j0k3r I'm working on the whole italian translation right now, I'll create a PR once it is all done, adding this new part, is that ok?

**Comment by j0k3r** [comment]
@matteocoder it's ok

**Comment by nicosomb** [comment]
OK for me. We can merge, :de: and :it: translations will come later.

**Pull request merged**